storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -621,6 +791,14 @@ Options
 
    The server instance being backed up.
 
+.. option:: --server-public-key-path
+
+   The file path to the server public RSA key in the PEM format.
+
+.. option:: --skip-tables-compatibility-check
+
+   See :option:`--tables-compatibility-check`.
```

Comment:
"skip" is actually just a prefix, it is equivalent of `--option=FALSE/OFF`

Reply:
Unless this rule applies everywhere and the user can use --skip-* version for each boolean option, I would suggest that we keep --skip-* options to improve the discoverability

Reply:
skip apply to *each* boolean option. https://dev.mysql.com/doc/refman/8.0/en/option-modifiers.html